src/lib/service/walletConnect.ts

```diff
@@ -13,7 +13,7 @@ import {
 import { StellarNetwork } from '../stellar/StellarNetwork';
 import { WCClient, WCModal } from './lib/walletconnect';
 
-export type WalletConnectNetwork = 'testnet' | 'public';
+export type WalletConnectNetwork = 'testnet' | 'public' | 'futurenet';
```

Comment:
If WalletConnect doesn't support futurenet network yet, why are we adding it?

Reply:
Because WalletConnect is assigned with Stellar Networks and gives types errors